code/controllers/subsystems/planets.dm

```diff
@@ -42,7 +42,7 @@ SUBSYSTEM_DEF(planets)
 			return
 		if(istype(T, /turf/unsimulated/wall/planetary))	
 			P.planet_walls += T
-		else if(istype(T, /turf/simulated) && T.outdoors)
+		else if(istype(T, /turf/simulated) && T.outdoors == OUTDOORS_YES)
```

Comment:
Should all of these checks use a ternary statement (probably make it a function it for simplicity, #defines might be messy) to check both turf and area outdoors where necessary?
i.e.,
```
/proc/is_outdoors(atom/A)
    var/turf/T = get_turf(A)
    if(istype(T))
         switch(T.outdoors)
            if(OUTDOORS_YES):
                return TRUE
            if(OUTDOORS_AREA):
                 var/area/A = get_area(T)
                 return A.outdoors
    return FALSE
```

Reply:
When a turf gets initialized and turned into an outdoor turf, even if it's by area, it's `outdoors` var is set to `OUTDOORS_YES`.
I now realize that I don't believe that occurs for default indoor turfs so that might need to get tweaked, or I can remove init turning `outdoors` into a binary value and instead keep `OUTDOORS_AREA` and use a proc, tho I'm not sure how many situations where keeping it a ternary after init would be useful.